CMakeLists.txt

```diff
@@ -101,9 +115,13 @@ add_library(libenrico
             ${SOURCES})
 
 set_target_properties(libenrico PROPERTIES OUTPUT_NAME enrico)
-target_link_libraries(libenrico PUBLIC iapws libnek5000 libopenmc xtensor heat_xfer pugixml gsl-lite)
+target_link_libraries(libenrico PUBLIC iapws libopenmc xtensor heat_xfer pugixml gsl-lite)
```

Comment:
I'm thinking it would be good to set a variable for the list of libraries found here (following with the `SOURCES` variable, maybe `LIBRARIES` could be used) and add `libnek5000` to that list conditionally depending on the state of `NEK_BUILD`. This would allow us to have only one call to `target_link_libraries` and makes the lines a little more readable IMO.

`target_link_libraries(libenrico PUBLIC ${ENRICO_LIBS})`

Reply:
So replace all current `target_link_libraries` calls with with setting or appending to a `LIBRARIES` variable and then link all libraries with one call? And will the location of `target_compile_definitions` for `libenrico` need to be changed to after the single `target_link_libraries` call?